encodings/roaring/src/boolean/mod.rs

```diff
@@ -32,22 +35,37 @@ impl RoaringBoolArray {
         if length < bitmap.cardinality() as usize {
             vortex_bail!("RoaringBoolArray length is less than bitmap cardinality")
         } else {
+            let roaring_stats = bitmap.statistics();
+            let stats = StatsSet::from(HashMap::from([
+                (
+                    Stat::Min,
+                    (roaring_stats.cardinality == length as u64).into(),
+                ),
+                (Stat::Max, (roaring_stats.cardinality > 0).into()),
+                (
+                    Stat::IsConstant,
+                    (roaring_stats.cardinality == length as u64 || roaring_stats.cardinality == 0)
+                        .into(),
+                ),
+                (Stat::TrueCount, roaring_stats.cardinality.into()),
+            ]));
+
             Ok(Self {
                 typed: TypedArray::try_from_parts(
                     DType::Bool(NonNullable),
                     length,
                     RoaringBoolMetadata { length },
-                    Some(Buffer::from(bitmap.serialize::<Portable>())),
+                    Some(Buffer::from(bitmap.serialize::<Native>())),
```

Comment:
we never care about Portable serialization since we will always read through one implementation